platform/o.n.bootstrap/src/org/netbeans/Module.java

```diff
@@ -605,6 +605,14 @@ final void assignInstrumentation(NbInstrumentation agent) {
     void unregisterInstrumentation() {
         NbInstrumentation.unregisterAgent(instr);
     }
+    
+    /**
+     * Release references to the ClassLoader. Package-private for now as only (?) FixedModule
+     * should retain the CL instance.
+     */
+    void releaseClassLoader() {
```

Comment:
Needed because I need to `null` classloader reference in fragments, yet do not want to bring their CL down using `classLoaderDown`.